api/app/routes/review.routes.ts

```diff
@@ -319,7 +320,7 @@ export function reviewRoutes(app: Express) {
   router.get("/", reviews.findAll);
 
   // Retrieve reviews for admin
-  router.get("/all", admin, reviews.getAllReviewsForAdmin);
+  router.get("/all", authorizeRoles([USER_PERMISSIONS.ADMIN, USER_PERMISSIONS.EVALUATOR]), reviews.getAllReviewsForAdmin);
```

Comment:
If we are using this here, why not remove the `admin` middleware and replace all its occurrences with `authorizeRoles`?
It doesn't make sense to keep two versions of the same functionality.

Reply:
I have modified the commit to reflect this change. Pls review again